contracts/TablelandTables.sol

```diff
@@ -149,6 +149,24 @@ contract TablelandTables is
         super._beforeTokenTransfer(from, to, tokenId);
     }
 
+    event TableTransfer(address from, address to, uint256 tableId);
+
+    function _afterTokenTransfer(
+        address from,
+        address to,
+        uint256 tokenId
+    )
+        internal
+        virtual
+        override(ERC721Upgradeable)
+        whenNotPaused
+    {
+        super._afterTokenTransfer(from, to, tokenId);
```

Comment:
So we end up with two events in the case that this is a normal transfer, correct?